src/mixins/enum-mixin.ts

```diff
@@ -223,13 +223,13 @@ export default class EnumMixin extends Vue {
    * @param alterationRequired A boolean indicating a required business type change
    * @returns the filing name
    */
-  filingTypeToName (type: FilingTypes, agmYear = null as string, alterationRequired = false): string {
```

Comment:
This method should be simple and not know about state if possible.

I created new filing type CHANGE_OF_COMPANY_INFO to handle this instead.